Tasks/Common/sonarqube-common/sonarqube-common.ts

```diff
@@ -3,12 +3,13 @@
 import tl = require('vsts-task-lib/task');
 import {ToolRunner} from 'vsts-task-lib/toolrunner';
 
-// Creates the tool runner for executing SonarQube.
-export function applySonarQubeParams(toolRunner:ToolRunner, sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?, sqDbUsername?, sqDbPassword?):ToolRunner {
-
+// Applies required parameters for connecting a Java-based plugin to SonarQube.
+// sqDbUrl, sqDbUsername and sqDbPassword are required if the SonarQube version is less than 5.2.
+export function applySonarQubeConnectionParams(toolRunner:ToolRunner, sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?, sqDbUsername?, sqDbPassword?):ToolRunner {
```

Comment:
Please add a remark to these functions stating that these arguments work for Maven and Gradle